docs/02.deploying/01.production/01.configmap/01.configmap.md

```diff
@@ -13,13 +13,46 @@ The 'always_reload: true' setting can be added in any ConfigMap yaml to force re
 
 #### Complete Sample NeuVector ConfigMap (initcfg.yaml)
 
-The latest ConfigMap can be found [here](https://raw.githubusercontent.com/neuvector/manifests/main/kubernetes/5.3.0/initcfg.yaml).
+The latest ConfigMap can be found [here](https://raw.githubusercontent.com/neuvector/manifests/main/kubernetes/5.4.0/initcfg.yaml).
 
 The sample is also shown below. This contains all the settings available. Please remove the sections not needed and edit the sections needed. Note: If using configmap in a secret, see section below for formatting changes.
 
 ```yaml
 apiVersion: v1
 data:
+  fedinitcfg.yaml: |
+    # ============ this section is used for primary cluster ============ >>>
+    # Optional. If specified, it overwrites the cluster name specified in the system configuration in the sysinitcfg.yaml file
+    Cluster_Name: primary.cluster.local
+    # Required and must be the same on primary cluster and remote clusters
+    # It must be 36 characters long, i.e., 32 hex characters grouped as 8-4-4-4-12 and separated by four hyphens
+    Join_Token: 2be93d8f-d42a-44fc-9d33-7c1a6153066b
+    # Required: REST server/port of the neuvector-svc-controller-fed-master service
+    Primary_Rest_Info:
+      Server: 1.2.3.4
+      Port: 11443
+    # Optional. Supported value: https
+    Use_Proxy: ""
+    # Optional. Whether federal repo scan data deployment is enabled (for primary cluster only)
+    Deploy_Repo_Scan_Data: false
+    # <<< ============ this section is used for primary cluster ============
+    # ============ this section is used for remote cluster ============ >>>
+    # Optional. If specified, it overwrites the cluster name specified in the system configuration in the sysinitcfg.yaml file
+    Cluster_Name: remote.cluster.local
+    # Required and must be the same on primary cluster and remote clusters
+    # It must be 36 characters long, i.e., 32 hex characters grouped as 8-4-4-4-12 and separated by four hyphens
```

Comment:
Please add a note: the Join_Token specified in the remote cluster's fedinitcfg.yaml needs to be the same as the Join_Token specified in the primary cluster's fedinitcfg.yaml
Otherwise the auto-joining request will be declined by the primary cluster.

Reply:
Updated in https://github.com/neuvector/docs/commit/150fcfa6f883e013091dc3098f80d9f5c261c08a